### PR TITLE
WIP - Merge rc2 to master

### PR DESCRIPTION
**Pull request by TravisEz13**
Merge 6.0.0 branch to master

### CODE REVIEW

**Comment by TravisEz13** [comment]
LGTM now

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,79 @@
   In the future, you should be able to add it by running `Install-Module Pester`. (#5623, #5631)
 - Make Travis CI use `libcurl+openssl+gssapi` (#5629, @markekraus)
 - Refactor `New-UnixPackaging` into functions to make the large function more readable. (#5625)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: This part needs to be removed.

File: docs/installation/linux.md [hunk 2]
```diff
@@ -657,19 +657,19 @@ pwsh
...
 ```
 
 ### Uninstallation - Kali
 
 ```sh
-dpkg -r powershell_6.0.0-rc-1.ubuntu.16.04_amd64.deb
+dpkg -r powershell_6.0.0-rc.2-1.ubuntu.16.04_amd64.deb
 ```
 
 ## Raspbian
```
Comment (daxian-dbw) [inline docs/installation/linux.md]: In the "Raspbian" section, I found 2 problems:
image

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@TravisEz13 will push the merge branch to master directly.

**Pull request closed without merging**